src/psygnal/_throttler.py

```diff
@@ -27,9 +30,18 @@ def __init__(
         self._args: tuple[Any, ...] = ()
         self._kwargs: dict[str, Any] = {}
 
+        # this mimics what functools.wraps does, but avoids __dict__ usage and other
+        # things that won't work with mypyc... HOWEVER, most of these dynamic
```

Comment:
By won't work, I guess you mean that mypyc will just ignore these? But won't error?

Reply:
correct.  `__doc__` will be `None` on mypyc... (i.e. it won't update the doc on the instance itself)